dashbuilder/modules/dashbuilder/module.yaml

```diff
@@ -0,0 +1,40 @@
+---
+schema_version: 1
+name: "rhpam-7-dashbuilder"
+version: "1.0"
+description: "Red Hat Dashbuilder 7.10 installer"
+labels:
+  - name: "org.jboss.product"
+    value: "rhpam-dashbuilder"
+  - name: "org.jboss.product.version"
+    value: "7.10.0"
+  - name: "org.jboss.product.rhpam-dashbuilder.version"
+    value: "7.10.0"
+envs:
+  - name: "JBOSS_PRODUCT"
+    value: "rhpam-dashbuilder"
+  - name: "RHPAM_DASHBUILDER_VERSION"
+    value: "7.10.0"
+  - name: "PRODUCT_VERSION"
+    value: "7.10.0"
+  - name: "ADD_ONS_DISTRIBUTION_ZIP"
+    value: "add_ons_distribution.zip"
+  - name: "DASHBUILDER_DISTRIBUTION_ZIP"
+    value: "rhpam-7.10.0-dashbuilder-runtime.zip"
+ports:
+  - value: 8080
+artifacts:
+  - name: "ADD_ONS_DISTRIBUTION_ZIP"
+    target: "add_ons_distribution.zip"
+    # rhpam-7.10.0.redhat-20201101-add-ons.zip
+    md5: "7e285242ded7b9c7f1b508830b1a10ae"
+run:
+  user: 185
```

Comment:
is this the UID which is needed by Dashbuilder to run?

Reply:
it is a pattern we've created in the past, this UDI refers to the "jboss" user.